bokeh/core/_templates/js_resources.html

```diff
@@ -14,7 +14,7 @@
 #}
 {%- for file in js_files %}
 <script type="text/javascript" src="{{ file }}"
-{%- if hashes %} integrity="sha384-{{ hashes[file] }}" crossorigin="anonymous"{%- endif -%}
+{%- if hashes and file in hashes %} integrity="sha384-{{ hashes[file] }}" crossorigin="anonymous"{%- endif -%}
```

Comment:
I suppose the slight risk here is that a pkg is published missing some hashes, and then this will just silently revert to the old hash-less way. Seems a fairly low risk though